ChangeLog.d/bugfix_1.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix:
+    * x509: do not include time.h without MBEDTLS_HAVE_TIME.
+    * Fixes type-limit error programs/ssl/ssl_context_info.c.
```

Comment:
Please use complete, grammatical sentences in the change log. Also, please write with the library user in mind. Why would the user care about a “type-limit error”? Does the compilation fail on some platforms? Is there a runtime bug?